### PR TITLE
Forward location to Flurry. Closes #112

### LINKED ISSUE #112: add support for flurry location optionally
http://support.flurry.com/index.php?title=Analytics/GettingStarted/TechnicalQuickStart#Track_Geographic_Location

### PR DESCRIPTION
**Pull request by f2prateek**

### CODE REVIEW

File: Analytics/Integrations/Flurry/SEGFlurryIntegration.m [hunk 1]
```diff
@@ -45,6 +45,16 @@ - (void)validate {
   self.valid = hasAPIKey;
 }
 
+#pragma mark - Utilities
+- (float)floatForKey:(NSDictionary *)dictionary:id:(float)defaultValue  {
```
Comment (f2prateek) [inline Analytics/Integrations/Flurry/SEGFlurryIntegration.m]: Is there a better way to get default values from a dictionary?

**Comment by f2prateek** [comment]
cc @travisjeffery

**Comment by travisjeffery** [comment]
implemented in https://github.com/segmentio/analytics-ios/pull/264, @f2prateek made some explanations there address your questions!

File: Analytics/Integrations/Flurry/SEGFlurryIntegration.m (hunk 1 above)
Comment (travisjeffery) [inline Analytics/Integrations/Flurry/SEGFlurryIntegration.m]: yep! checkout: https://github.com/segmentio/analytics-ios/pull/264

**Pull request closed without merging**